From b0448d7816ca561b6470dc50fc1621aec2620f57 Mon Sep 17 00:00:00 2001
From: Fathi Boudra <fathi.boudra@linaro.org>
Date: Wed, 22 Mar 2017 17:40:11 +0200
Subject: [PATCH] selftests: sigaltstack: fix packaging

sigaltstack was not using TEST_PROGS resulting in binary not being installed.

Signed-off-by: Denys Dmytriyenko <denys@ti.com>
---
 tools/testing/selftests/sigaltstack/Makefile |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

--- a/tools/testing/selftests/sigaltstack/Makefile
+++ b/tools/testing/selftests/sigaltstack/Makefile
@@ -1,8 +1,8 @@
 CFLAGS = -Wall
-BINARIES = sas
-all: $(BINARIES)
+TEST_PROGS = sas
+all: $(TEST_PROGS)
 
 include ../lib.mk
 
 clean:
-	rm -rf $(BINARIES)
+	rm -rf $(TEST_PROGS)
